### PR TITLE
added blog card into single component 

### LINKED ISSUE #124: Home page blog: add links to each blog item
- [ ] add links to each blog item

- [ ] check shadow for text to be clear and readable

- [ ] check the responsiveness of the carousel

image

### PR DESCRIPTION
**Pull request by hiba-machfej**
In this PR I added the blog card into a single component and used it in the blog's carousel on the home page instead of the dummy data it was there. This also closes #124
Screen Shot 2020-11-22 at 12 09 34 PM
Screen Shot 2020-11-22 at 12 09 51 PM
Screen Shot 2020-11-22 at 12 10 05 PM
Screen Shot 2020-11-22 at 12 10 27 PM

### CODE REVIEW

File: src/components/BlogCard/index.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import "../../style/variables";
+
+.blogPost {
+  height: 20rem;
+  border-radius: 1rem !important;
```
Comment (Jamil-Bailony) [inline src/components/BlogCard/index.scss:5]: should be the variable that we use for border radius

Comment (Jamil-Bailony) [reply src/components/BlogCard/index.scss:5]: if that not suitable, I suggest to create one for image and use it

File: src/components/BlogCard/index.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    color: $primaryColor;
+  }
+
+  &:hover {
+    text-decoration: underline;
+    color: $primaryColor;
+  }
+
+  a {
+    color: $primaryColor;
```
Comment (Jamil-Bailony) [inline src/components/BlogCard/index.scss]: this one is repeated three times. I suggest to refactor it, give it to .blogPost for example and set this to `inherit` or try smth else

File: src/components/BlogCard/index.scss (hunk 1 above)
Comment (hiba-machfej) [reply src/components/BlogCard/index.scss:5]: it looks really small and not like the other cards, the border-radius depends on the size of the element :


Screen Shot 2020-11-22 at 12 25 01 PM

Screen Shot 2020-11-22 at 12 25 07 PM

Screen Shot 2020-11-22 at 12 25 04 PM

File: src/components/BlogCard/index.scss (hunk 2 above)
Comment (hiba-machfej) [reply src/components/BlogCard/index.scss]: ok done it actually works without it

File: src/components/BlogCard/index.scss (hunk 1 above)
Comment (hiba-machfej) [reply src/components/BlogCard/index.scss:5]: the first image I used radius in the variables

File: src/components/BlogCard/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { BrowserRouter as Router } from "react-router-dom";
+import BlogCard from "./index";
+
+const mockData = {
```
Comment (jennetturkkan) [inline src/components/BlogCard/index.test.jsx]: make it MOCK_DATA

File: src/components/BlogCard/index.scss (hunk 1 above)
Comment (Jamil-Bailony) [reply src/components/BlogCard/index.scss:5]: you can create one for all cards and images, so we can control it in one place

**Review by jennetturkkan — APPROVED** [review]

File: src/components/BlogCard/index.test.jsx (hunk 3 above)
Comment (hiba-machfej) [reply src/components/BlogCard/index.test.jsx]: Done

**Review by louisrli — APPROVED** [review]
this PRs a little big for post code freeze but its a straightforward feature i suppose...test it well

**Comment by jennetturkkan** [comment]
@louisrli it works good. we tested it. is it okay if we merge?

**Pull request merged**